Code/Diversity_Histogram.py

```diff
@@ -65,15 +70,19 @@
                                                         Gene_Positive_Negative_Number['Total_Diauxic']/38)\
                                                         /(Gene_Positive_Negative_Number['Total_Diauxic']/38+\
                                                          Gene_Positive_Negative_Number['Total_Polarity']/123)
-
+# questions: 
+    # - where do 123 and 38 come from? 
+    # - can you explain this expresion you used to compute the ['polarity_over_diauxic']? 
```

Comment:
polarity_over_diauxic is a new column in the **Gene_Positive_Negative_Number** dataframe that gives the relative ratio of the interactions each gene has (**Polarity-Diauxic**) over its total number of interactions. The numbers 123 and 38 is to correct for the difference in the number of genes in the two modules